src/patternfly/components/Accordion/examples/Accordion.md

```diff
@@ -294,6 +295,62 @@ cssPrefix: pf-v5-c-accordion
 {{/accordion}}
 ```
 
+### Toggle icon at start
+
+```hbs
+{{#> accordion accordion--IsStartAligned="true"}}
+  {{#> accordion-toggle}}
+    {{#> accordion-toggle-icon}}{{/accordion-toggle-icon}}
+    {{#> accordion-toggle-text}}Item one{{/accordion-toggle-text}}
+  {{/accordion-toggle}}
+  {{#> accordion-expandable-content}}
+    {{#> accordion-expandable-content-body}}
+      This text is hidden
+    {{/accordion-expandable-content-body}}
+  {{/accordion-expandable-content}}
+
+  {{#> accordion-toggle}}
+    {{#> accordion-toggle-icon}}{{/accordion-toggle-icon}}
+    {{#> accordion-toggle-text}}Item two{{/accordion-toggle-text}}
+  {{/accordion-toggle}}
+  {{#> accordion-expandable-content}}
+    {{#> accordion-expandable-content-body}}
+      This text is hidden
+    {{/accordion-expandable-content-body}}
+  {{/accordion-expandable-content}}
+
+  {{#> accordion-toggle}}
+    {{#> accordion-toggle-icon}}{{/accordion-toggle-icon}}
+    {{#> accordion-toggle-text}}Item three{{/accordion-toggle-text}}
+  {{/accordion-toggle}}
+  {{#> accordion-expandable-content}}
+    {{#> accordion-expandable-content-body}}
+      This text is hidden
+    {{/accordion-expandable-content-body}}
+  {{/accordion-expandable-content}}
+
+  {{#> accordion-toggle accordion-toggle--IsExpanded="true"}}
```

Comment:
Just in case you want to try and update any of these other pre-existing params to use `true`, one sneaky little thing to look for is to make sure we aren't relying on the string value to do something somewhere - eg, to do something like below. There's no harm in just leaving them the way they are now, either.

```
# Some .hbs file
<button
  aria-expanded="{{accordion-toggle--IsExpanded}}"
  ...
```